Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -425,21 +385,16 @@ - (CGSize)cellSizeWithTransaction:(YapDatabaseReadTransaction *)transaction
     return cellSize;
 }
 
-- (CGFloat)dateHeaderStrokeThickness
+- (CGFloat)dateHeaderVMargin
 {
-    return CGHairlineWidth();
-}
-
-- (CGFloat)dateHeaderBottomMargin
-{
-    return 20.f;
+    return 23.f;
 }
 
 - (CGFloat)dateHeaderHeight
 {
     if (self.viewItem.shouldShowDate) {
-        CGFloat textHeight = self.dateHeaderFont.capHeight;
-        return (CGFloat)ceil(self.dateHeaderStrokeThickness + textHeight + self.dateHeaderBottomMargin);
+        CGFloat textHeight = self.dateHeaderFont.lineHeight;
+        return (CGFloat)ceil(textHeight + self.dateHeaderVMargin * 2);
```

Comment:
I thought we were converging on an approach wherein the cell contents began at the cell's edge and intra-cell spacing was a concern of `ConverasationViewItem`?

Reply:
Ok - I've done this, but it's a little awkward because the design wants symmetrical spacing above and below this header. 

So now, the cell adds the bottom spacing while the conversationViewItem adds the equivalent amount of top spacing.

Reply:
Thanks - IMO it'll make it easier to do the right thing.